### ISSUE TITLE
Custom CSS :: Incorrect handling of ISO character definition

### ISSUE THREAD

**Issue by dm-chechik** [elegantthemes/Divi-Beta#840]
### Problem:

The CSS rules like this 
`div::after {
  content : '\61';
}`
are incorrectly handled.
The **\\** symbol is removed after saving.

## 📎 Attached PR

- https://github.com/elegantthemes/submodule-builder/pull/1521

**Comment by fikrirasyid** [comment]
This is pretty much like https://github.com/elegantthemes/Divi-Beta/issues/837

**Comment by andrey-agp** [comment]
Same issue with Custom CSS box in epanel. 
@fikrirasyid Should I open new issue for this?

**Comment by fikrirasyid** [comment]
@andrey-agp yeah, please open new issue for this. Once this solution is approved, I think we can implement it on ePanel and track it on different issue.

**Comment by lots0logs** [comment]
@fikrirasyid Could you check this one with changes from latest master pulled in? I'm getting a javascript error when trying to open any of the module settings in the BB:

![2016-12-16_01-47-10-pm](https://cloud.githubusercontent.com/assets/4675662/21276283/5497cb06-c396-11e6-9fd7-33cf9a1663ed.png)

**Comment by fikrirasyid** [comment]
@lots0logs I've fixed the issue via https://github.com/elegantthemes/submodule-builder/commit/84b1c28dc3b12cedaa6fb6b6043f6c9bec1b068b. It happens due to missing backslash in our JS which is printed in php.
